packages/labs/src/components/tag-input/_tag-input.scss

```diff
@@ -9,39 +9,56 @@
 @import "~@blueprintjs/core/src/components/forms/common";
 @import "~@blueprintjs/core/src/components/tag/common";
 
-$tag-input-padding: ($pt-input-height - $tag-height) / 2;
+$ti-padding: ($pt-input-height - $tag-height) / 2;
+$ti-right-padding: ($pt-input-height - $pt-button-height-small) / 2;
```

Comment:
So much nicer with this added 👏 Perfectly squared off. `$tag-input-` ain't that long is it?

How about `-right-padding` -> `-padding-right` to match the property name?